tests/runner.py

```diff
@@ -1693,13 +1686,13 @@ def build_library(name,
                              cwd=project_dir)
     except subprocess.CalledProcessError:
       with open(os.path.join(project_dir, 'configure_out')) as f:
-        print('-- configure stdout --')
-        print(f.read())
-        print('-- end configure stdout --')
+        logger.info('-- configure stdout --')
+        logger.info(f.read())
+        logger.info('-- end configure stdout --')
```

Comment:
I think I'd like to see how this kind of things looks in the output, before and after this change.   

In general I'm still feeling like this might not be worth the change.  But I'll check it out.

Reply:
Alright, let me know if you need any changes from me.